test-network-function/generic/suite.go

```diff
@@ -122,6 +128,87 @@ func createPartnerContainers(config *tnfConfig.TestConfiguration) map[tnfConfig.
 	return createContainers(config.PartnerContainers)
 }
 
+func pipeCmdToStd(cmd *exec.Cmd) {
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+}
+
+func waitForPod() error {
+	for i := 0; i < 10; i++ {
+		checkPodStatusCmd := exec.Command("oc", "get", "pod", "node-master", "-o", "jsonpath='{.status.containerStatuses[0].ready}'")
+		var buffer bytes.Buffer
+		bufferWriter := bufio.NewWriter(&buffer)
+		checkPodStatusCmd.Stdout = bufferWriter
+		err := checkPodStatusCmd.Run()
+		if err != nil {
+			return err
+		}
+		cmdResult := buffer.String()
+		if cmdResult == "'true'" {
+			return nil
+		}
+		time.Sleep(time.Second * 2)
+	}
+	return errors.New("Passed timeout for pod to be ready")
+}
+
+func createPrivilagedPod() error {
```

Comment:
This isn't needed.  The idea is not to create a pod that will fail, but to test the CNF pods our partners provide.

Reply:
This creates a privilaged pod on the node of the test pod in order to retrieve from the host filesystem the diff between the test container and it's corresponding image.

Reply:
So you are injecting an additional test Pod, gotcha.  Okay, then we should put that Pod definition in https://github.com/test-network-function/cnf-certification-test-partner.